invite/models.py

```diff
@@ -97,6 +97,8 @@ def send(self, request=None):
 class PasswordResetInvitation(AbstractInvitation):
     """Defines a model for invitations created for password resets."""
 
+    created = models.DateTimeField(auto_now_add=True, null=True)
+
```

Comment:
I'm thinking that, the `date_invited` field refers to the timestamp when an invitation link is sent as opposed to the password reset link and thought we could have a separate `created` field for it.

Reply:
It may be that the `date_invited` name is misleading because initially there was not a base class shared by `Invitation` and `PasswordResetInvitation`, so it was named and given help text that is more relevant to a new `Invitation`, however, I believe in the case of instantiating a new `PasswordResetInvitation`, `date_invited` and `created` basically end up being the same thing. Think of it as the date someone is invited to reset their password? :D

Reply:
After I tried using `date_invited` I realized that it is a `DateField` while I think using `DateTimeField` is appropriate here.